system/jlib/jatomic.hpp

```diff
@@ -40,6 +42,30 @@ inline static void spinPause() { } // MORE: Is there an equivalent?
 inline static void spinPause() { }
 #endif
 
+//Use this class for stats which are gathered, but the values read from other threads do not need to be synchronized
```

Comment:
Is it possible with this class to "lose" counts? If so that should be made clear in the comments